sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/SarifParserCallbackImpl.java

```diff
@@ -235,11 +232,6 @@ private static NewIssueLocation createIssueLocation(InputFile inputFile, Locatio
 
     } catch (IllegalArgumentException ex1) {
 
-      if (!isLocationResilient) {
-        // Our rules should fail if they report on an invalid location
-        throw ex1;
-      }
-
```

Comment:
I think it'd be good to still throw when it's our own rules (from `csharpsquid` repo), since its ours, we know that their location should be correct, and it's nice to fail in that case, to detect as early as possible any issue in how we're handling location